src/ansys/mapdl/core/_commands/post26/special_purpose.py

```diff
@@ -0,0 +1,588 @@
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+class SpecialPurpose:
+
+    def pmgtran(
+        self,
+        fname: str = "",
+        freq: str = "",
+        fcnam1: str = "",
+        fcnam2: str = "",
+        pcnam1: str = "",
+        pcnam2: str = "",
+        ecnam1: str = "",
+        ccnam1: str = "",
+        **kwargs,
+    ):
+        r"""Summarizes electromagnetic results from a transient analysis.
+
+        Mechanical APDL Command: `PMGTRAN <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_PMGTRAN.html>`_
+
+        Parameters
+        ----------
+        fname : str
+            File name (8 characters maximum) to which tabular data and plot files will be written. Must be
+            enclosed in single quotes when the command is manually typed in. Defaults to MG_TRNS. The data
+            file extension is :file:`.OUT` and the plot file extension is. :file:`PLT`.
+
+        freq : str
+            Frequency of solution output. Defaults to 1. Every ``FREQ`` th solution on the results file is
+            output.
+
+        fcnam1 : str
+            Names of element components for force calculation. Must be enclosed in single quotes when the
+            command is manually typed in.
+
+        fcnam2 : str
+            Names of element components for force calculation. Must be enclosed in single quotes when the
+            command is manually typed in.
+
+        pcnam1 : str
+            Names of element components for power loss calculation. Must be enclosed in single quotes when
+            the command is manually typed in.
+
+        pcnam2 : str
+            Names of element components for power loss calculation. Must be enclosed in single quotes when
+            the command is manually typed in.
+
+        ecnam1 : str
+            Names of element components for energy and total current calculations, respectively. Must be
+            enclosed in single quotes when the command is manually typed in.
+
+        ccnam1 : str
+            Names of element components for energy and total current calculations, respectively. Must be
+            enclosed in single quotes when the command is manually typed in.
+
+        Notes
+        -----
+
+        .. _PMGTRAN_notes:
+
+        :ref:`pmgtran` invokes a Mechanical APDL macro which calculates and summarizes electromagnetic
+        results from
```

Comment:
weird line break.

Reply:
Added in https://github.com/ansys/pyconverter-xml2py/issues/413#issue-3020603226